Automattic/themes
# Varia: Fix responsive logic for better mobile view

Fixes responsive logic to prevent content/text from reaching the absolute edge of any container or viewport edge. Effects Group Block, Cover Block, Pullquote Block, and Gallery Blocks.

#### Changes proposed in this Pull Request:

- Revises responsive logic Sass rules to allow for smarter padding on small screens. 
- Focuses the responsive logic on `.alignwide` and `.alignfull` since normal width have the correct width already. 
- Sets 50% max-width on `.alignright` and `.alignleft` Gallery Blocks.

#### Before:

![varia-align-broke](https://user-images.githubusercontent.com/709581/61829998-2265a880-ae38-11e9-855c-9b354f859b75.gif)

#### After:

![varia-align-fix](https://user-images.githubusercontent.com/709581/61829892-e29ec100-ae37-11e9-970d-32ebe44a2c40.gif)

#### To test:

- Try the Group Block, Cover Block, Pullquote Block, and Gallery Blocks and make sure the text contained within them never touches the edge of the screen or the widest edge of its parent container.

#### Related issue(s):

Found while testing locally.
